x-pack/plugins/security_solution/public/detections/components/rules/suppression_info_icon/index.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import { EuiLink, EuiPopover, EuiText, EuiButtonIcon } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import { useKibana } from '../../../../common/lib/kibana';
+
+const POPOVER_WIDTH = 300;
+
+/**
+ * Icon and popover that gives hint to users how suppression for missing fields work
+ */
+const SuppressionInfoIconComponent = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const { docLinks } = useKibana().services;
+
+  const onButtonClick = () => setIsPopoverOpen(!isPopoverOpen);
+  const closePopover = () => setIsPopoverOpen(false);
+
+  const button = (
+    <EuiButtonIcon
+      iconType="questionInCircle"
+      onClick={onButtonClick}
+      aria-label="Open help popover"
+    />
+  );
+
+  return (
+    <EuiPopover button={button} isOpen={isPopoverOpen} closePopover={closePopover}>
+      <EuiText style={{ width: POPOVER_WIDTH }}>
```

Comment:
nit: Text size feels a bit big in the popover at `16px` IMO, especially since the regular text below is at `14px`. To be honest though, we're not consistent here with our rich text 'tooltips', so could go either way here.

Suppression `16px`
<img width="300" alt="image" src="https://user-images.githubusercontent.com/2946766/236559713-73e9cfa4-7498-4101-a45d-0301bb4ebdf5.png">


Execution Log `14px`
<img width="300" alt="image" src="https://user-images.githubusercontent.com/2946766/236559306-c6f6a1a2-9f3b-4a97-bb1f-f39a1dc3389a.png">

Integrations `14px`
<img width="300" alt="image" src="https://user-images.githubusercontent.com/2946766/236559401-b543badf-5b7e-4d12-946f-923c996495c9.png">

Rule Monitoring `16px`
<img width="300" alt="image" src="https://user-images.githubusercontent.com/2946766/236559439-aba42bbf-1f39-455f-9a39-13c1a3e395d1.png">